weave/integrations/openai/gpt_image_utils.py

```diff
@@ -0,0 +1,215 @@
+from typing import Any, Callable, Optional
+
+from weave.integrations.openai.openai_utils import (
+    create_basic_wrapper_async,
+    create_basic_wrapper_sync,
+    maybe_unwrap_api_response,
+)
+from weave.trace.autopatch import OpSettings
+from weave.trace.weave_client import Call
+
+# Default model and size for DALL-E 2
+DEFAULT_IMAGE_MODEL = "dall-e-2"
+DEFAULT_IMAGE_SIZE = "1024x1024"
+
+
+def _postprocess_inputs_with_defaults(inputs: dict[str, Any]) -> dict[str, Any]:
+    """Returns the inputs, setting default model and size if not present."""
+    result = dict(inputs) if inputs is not None else {}
+    if "model" not in result or not result["model"]:
+        result["model"] = DEFAULT_IMAGE_MODEL
```

Comment:
this might be more cleanly expressed as:
```suggestion
    result["model"] = result.get("model", DEFAULT_IMAGE_MODEL)
```

Same below